docs/contrib/Logger.rst

```diff
@@ -298,7 +433,7 @@ ioPlacer         ppl
 OpenDB           odb
 opendp           dpl
 OpenRCX          rcx
-*OpenROAD*       ord
+OpenROAD         ord
```

Comment:
OpenROAD was marked to indicate that it is the application as a whole rather than a tool within the app.

Reply:
Marked this in italics-is this ok?